context/app/static/js/components/Providers.jsx

```diff
@@ -14,24 +15,58 @@ const generateClassName = createGenerateClassName({
   seed: 'portal',
 });
 
+// siteIds should correspond to the list at:
+// https://hubmapconsortium.matomo.cloud/index.php?module=SitesManager
+
+// eslint-disable-next-line no-restricted-globals
+const { host } = location;
+let siteId = 3;
+if (host === 'portal.hubmapconsortium.org') {
+  siteId = 1;
+} else if (host === 'localhost:5001') {
+  siteId = 2;
+}
+
+const instance = createInstance({
+  urlBase: 'https://hubmapconsortium.matomo.cloud/',
+  siteId,
+  // userId: 'UID76903202', // optional, default value: `undefined`.
+  // trackerUrl: 'https://LINK.TO.DOMAIN/tracking.php', // optional, default value: `${urlBase}matomo.php`
+  // srcUrl: 'https://LINK.TO.DOMAIN/tracking.js', // optional, default value: `${urlBase}matomo.js`
+  // disabled: false, // optional, false by default. Makes all tracking calls no-ops if set to true.
+  // heartBeat: { // optional, enabled by default
+  //   active: true, // optional, default value: true
+  //   seconds: 10 // optional, default value: `15
+  // },
+  // linkTracking: false, // optional, default value: true
+  // configurations: { // optional, default value: {}
+  //   // any valid matomo configuration, all below are optional
+  //   disableCookies: true,
+  //   setSecureCookie: true,
+  //   setRequestMethod: 'POST'
+  // }
+});
+
```

Comment:
I'd pull `createInstance` and the provider into a separate component and the `siteId` logic into a util with a case switch to avoid the global and `let`.

Reply:
Good call; Should look better now.